packages/core/src/internal/proc.js

```diff
@@ -163,15 +169,18 @@ export default function proc(env, iterator, parentContext, parentEffectId, meta,
       cb.cancel = noop // defensive measure
       if (env.sagaMonitor) {
         if (isErr) {
-          env.sagaMonitor.effectRejected(effectId, res)
+          env.sagaMonitor.effectRejected(effectId, SagaError.isErrorAlreadyWrapped(res) ? res.getError() : res)
```

Comment:
when this can be not wrapped? shouldnt the wrapping happen right away after catching the error? so it should be already wrapped when it reaches this line?

Reply:
Chungtran6655@gmail.com

Reply:
ok, so it seems this gets actually called before wrapping as `currCb` in which we are here is actually the first thing to be called by effect runners 😅